element_volume/volume_matching.py

```diff
@@ -0,0 +1,129 @@
+import datajoint as dj
+import numpy as np
+import hashlib
+import uuid
+
+from element_interface.utils import dict_to_uuid
+
+from . import volume
+
+
+schema = dj.Schema()
+
+_linking_module = None
+
+
+def activate(
+    schema_name: str,
+    *,
+    create_schema: bool = True,
+    create_tables: bool = True
+):
+    """Activate this schema.
+    Args:
+        schema_name (str): Schema name on the database server to activate the
+            `imaging_report` schema
+        create_schema (bool): When True (default), create schema in the database if it
+            does not yet exist.
+        create_tables (bool): When True (default), create tables in the database if they
+            do not yet exist.
+    """
+    assert volume.schema.is_activated(), 'The "volume" schema must be activated'
+    schema.activate(
+        schema_name,
+        create_schema=create_schema,
+        create_tables=create_tables,
+        add_objects=volume.__dict__,
+    )
+
+
+@schema
+class VolumeMatchTask(dj.Manual):
+    definition = """  # 
+    volume_match_task: uuid
+    """
+
+    class Volume(dj.Part):
+        definition = """
+        -> master
+        -> volume.Segmentation
+        """
+
+    @classmethod
+    def insert1(cls, vol_keys, **kwargs):
+        """
+        Args:
+            vol_keys (tuple): a tuple of (volA_key, volB_key)
+        """
+        assert len(vol_keys) == 2, f"Volume match task only support matching two volumes, {len(vol_keys)} are provided"
+        vol_keys = [(volume.Segmentation & k).fetch1("KEY") for k in vol_keys]
+        assert len(set(vol_keys)) == 2, "The two specified volumes are identical"
```

Comment:
I would suggest that we request the user pass in the keys of the `volume.Segmentation` table to handle the case where multiple segmentations are run on the same volume.